features/buyer/buyer_create_briefs.feature

```diff
@@ -20,7 +20,8 @@ Scenario: Start brief creation for an individual specialist
   When I choose 'North East England' for 'location'
   And I click 'Save and continue'
   Then I should be on the "Overview of work" page for the buyer brief 'Find an individual specialist brief'
-  @wip
+
+@wip
```

Comment:
I don't think this needs to be `@wip` any more - the buyer dashboard does list briefs now, so this test should run OK:
![screen shot 2016-03-15 at 10 49 47](https://cloud.githubusercontent.com/assets/6525554/13775294/b411e690-ea9b-11e5-8873-5ff58a46556e.png)